Bottom: 08cdb9a8cfd429d8b0018d91813c88ace4431342
Top:    c27d867ad4aa3272562653403c7f0f0398a62b7c
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2013-06-10 10:56:18 -0700

Refresh of use_ib_acm

---

diff --git a/src/acm.c b/src/acm.c
index c423bb7..c9ca5b5 100755
--- a/src/acm.c
+++ b/src/acm.c
@@ -45,6 +45,7 @@
 #include <infiniband/ib.h>
 #include <infiniband/sa.h>
 
+#ifdef USE_IB_ACM
 #include <infiniband/acm.h>
 
 #if DEFINE_ACM_MSG
@@ -400,3 +401,5 @@ void ucma_ib_resolve(struct rdma_addrinfo **rai, struct rdma_addrinfo *hints)
 	if (af_ib_support && !(hints->ai_flags & RAI_ROUTEONLY) && (*rai)->ai_route_len)
 		ucma_resolve_af_ib(rai);
 }
+
+#endif /* USE_IB_ACM */
diff --git a/src/cma.h b/src/cma.h
index e6fba8b..b575b9b 100644
--- a/src/cma.h
+++ b/src/cma.h
@@ -162,9 +162,15 @@ extern int af_ib_support;
 
 #define RAI_ROUTEONLY		0x01000000
 
+#ifdef USE_IB_ACM
 void ucma_ib_init();
 void ucma_ib_cleanup();
 void ucma_ib_resolve(struct rdma_addrinfo **rai, struct rdma_addrinfo *hints);
+#else
+#define ucma_ib_init()
+#define ucma_ib_cleanup()
+#define ucma_ib_resolve(x, y)
+#endif
 
 /* Define path record definition if using older version of libibverbs */
 #ifdef DEFINE_PATH_RECORD
